src/wallet/rpcwallet.cpp

```diff
@@ -3774,8 +3774,8 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
             "}\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("getaddressinfo", EXAMPLE_ADDRESS) +
-                    HelpExampleRpc("getaddressinfo", EXAMPLE_ADDRESS)
+                    HelpExampleCli("getaddressinfo", EXAMPLE_ADDRESS[1]) +
```

Comment:
The quotation marks around the address are missing (see comment to `"validateaddress"` above).